hudi-common/src/main/java/org/apache/hudi/common/table/log/AbstractHoodieLogRecordReader.java

```diff
@@ -120,28 +120,32 @@
   private int totalScannedLogFiles;
   // Progress
   private float progress = 0.0f;
+  // Partition name
+  private Option<String> partitionName;
+  // Virtual keys
+  private boolean virtualKeysEnabled = false;
```

Comment:
why introduce another construct like `virtual keys` here? just stick to using `populateMetaFields` and go about doing our thing?

Reply:
sure, retaining the old name `populateMetaFields`